table/block_based_table_builder.cc

```diff
@@ -1064,7 +1064,7 @@ void BlockBasedTableBuilder::EnterUnbuffered() {
   if (!r->data_block_and_keys_buffers.empty()) {
     while (compression_dict_samples.size() < kSampleBytes) {
       size_t rand_idx =
-          generator.Uniform(r->data_block_and_keys_buffers.size());
+          (size_t)generator.Uniform(r->data_block_and_keys_buffers.size());
```

Comment:
use C++ style casting `static_cast<>`. Same in other places.
We are following Google C++ Style and it is the suggestion there: https://google.github.io/styleguide/cppguide.html#Casting

Reply:
Ok, I have updated the coding style.